apps/mobile-wallet/src/components/RefreshSpinner.tsx

```diff
@@ -18,3 +19,25 @@ const RefreshSpinner = (props: Partial<RefreshControlProps>) => {
 }
 
 export default RefreshSpinner
+
+const useRefreshAddressesBalances = () => {
```

Comment:
Do you plan to leave it in the RefreshSpinner component? Or have you a masterplan and you want to move this later on?